### DISCUSSION TITLE
Some actions like "Add all missing imports" don't work

### DISCUSSION THREAD

**Discussion by codentacos** [zed-industries/zed#14937] [Feature Requests]
Does anybody else have issues with these buttons not doing anything?
<img width="786" alt="Xnapper-2024-07-21-19 54 01" src="https://github.com/user-attachments/assets/a2a15697-3984-4f1d-bade-42097472be29">

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
